wct.conf.json

```diff
@@ -25,7 +25,7 @@
         {
           "browserName": "safari",
           "platform": "OS X 10.13",
-          "version": ""
+          "version": "11.1"
```

Comment:
I think the version was previously remove to get past some issues, and I think there are other repos where it is also blank.  Curious to know more about this.

Reply:
This is a short term solution until sauce lab fix their issue that default safari version 12 is missing. https://d2l.slack.com/archives/C5EEEFMHV/p1539287663000100.